editor/store/actions.js

```diff
@@ -10,7 +10,7 @@ import { partial, castArray } from 'lodash';
  *
  * @param  {Object} post     Post object
  * @param  {Object} settings Editor settings object
- * @return {Object}          Action object
+ * @returns {Object}         Action object
```

Comment:
ToDo: Add new line before `@returns` and remove excess spacing

Reply:
@aduth This was the only instance that I missed I found, did you come across any others?